src/views/Dashboard/Customer/Home/RequestTest/PaymentDetailCard.vue

```diff
@@ -312,11 +308,6 @@ export default {
 
       if(this.lastOrder){
         this.detailOrder = await queryOrderDetailByOrderID(this.api, this.lastOrder)
-
-        if (this.detailOrder.status === "Unpaid") {
-          this.showAlert = true
-          return
-        }
```

Comment:
Are you sure about this?

Reply:
yes this condition should be only on step 2, when customer click select service